### PR TITLE
Adding UniFi OS

### PR DESCRIPTION
**Pull request by Darkiros**
Adding UniFi OS version detection.

Exemple of website : 
https://212.83.41.101:8443/manage/account/login
https://178.238.148.149:8443/manage/account/login
https://13.68.128.84:8443/manage/account/login

### CODE REVIEW

File: src/technologies/u.json [hunk 1]
```diff
@@ -65,6 +65,23 @@
...
+    "cats": [
+      28
+    ],
+    "description": "UniFi OS is the operating system for UniFi products, which provides a user interface.",
+    "dom": {
+      "div[class*='unifiAccount__appNameAndVersion']": {
+        "text": "([\\d+\\.]+)\\;version:\\1"
+      }
+    },
+    "implies": "AngularJS",
```
Comment (nurbek91) [inline src/technologies/u.json:78]: For performance, it may be better to replace with requires?

File: src/technologies/u.json [hunk 2]
```diff
@@ -65,6 +65,23 @@
...
+  "UniFi OS": {
+    "cats": [
+      28
+    ],
+    "description": "UniFi OS is the operating system for UniFi products, which provides a user interface.",
+    "dom": {
+      "div[class*='unifiAccount__appNameAndVersion']": {
+        "text": "([\\d+\\.]+)\\;version:\\1"
+      }
+    },
```
Comment (nurbek91) [inline src/technologies/u.json:77]: These things seem to duplicate js. Maybe it's better to leave only js?
unifiConstant.VERSION

**Review by nurbek91 — COMMENTED** [review]
Hey mate, thanks for PR. just leave comments at your discretion

File: src/technologies/u.json (hunk 1 above)
Comment (Darkiros) [reply src/technologies/u.json:78]: Okay, i will change it for "require".

File: src/technologies/u.json (hunk 2 above)
Comment (Darkiros) [reply src/technologies/u.json:77]: Yes, but it is not the same thing. I put several sources for the detection. 
I can remove the dom detection if needed.

Comment (nurbek91) [reply src/technologies/u.json:77]: I checked in the examples you provided and it works in all js. If there are examples where there is no js then of course you can leave

Comment (Darkiros) [reply src/technologies/u.json:77]: Yes I check on other websites and it works without dom. I remove the dom part.

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**